builder.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker run --rm -v "${PWD}:/go/src/kubevirt.io/kubevirt" -w "/go/src/kubevirt.io/kubevirt" -v "${HOME}/.glide:/home/user/.glide" -e USER_ID=$(id -u) -e GROUP_ID=$(id -g) -it kubevirt/builder "$@"
```

Comment:
In general, do we want project dependencies to be on the dev machine?
I see benefit in both, having it inside of the container and having it on the dev machine.
The first would lead to more stable builds, the second for more flexible dev environment.

I am tempted to vote for the stable one, to create single stable environment for each developer to work with.